src/modules/emote_menu/components/EmoteMenuPopover.jsx

```diff
@@ -20,20 +25,29 @@ const EmoteMenuPopover = React.forwardRef(
       setTip(show);
     }
 
-    useAutoPositionPopover(localRef, boundingQuerySelector, style, hasTip);
+    const width = useHorizontalResize(emoteMenuWidth, boundingQuerySelector, 300, handleRef);
+    useAutoPositionPopover(localRef, boundingQuerySelector, style, hasTip, width);
+
+    React.useEffect(() => {
+      setEmoteMenuWidth(width);
```

Comment:
how does this work with different sized chat windows, like popout chat vs normal chat? won't this cause your setting to be overwritten?